api/models/Tag.js

```diff
@@ -15,7 +15,7 @@ var addToTaggable = (taggable, tagName, selected, trx) => {
     association = 'post.communities'
     communities = comment => comment.relations.post.relations.communities.models
   }
-  return taggable.load(association)
+  return taggable.load(association, {transacting: trx})
   .then(() => Tag.find(tagName))
   .then(tag => {
     if (tag) {
```

Comment:
@levity there's a race condition here (if two people create the tag at the same time). Do you have any pointers on the best way to handle this/other places in the code where you've handled it?